src/main/java/graphql/introspection/IntrospectionWithDirectivesSupport.java

```diff
@@ -247,7 +270,7 @@ public GraphQLDirective getDirective() {
         /**
          * @return the directive to be included
          */
-        GraphQLDirective getDirective();
+        String getDirectiveName();
```

Comment:
BREAKING CHANGE!!

We could put in a shimand defaulted method but do we want to?  I dont think its use enough to warrant it

Thoughts?